### PR TITLE
Few cleanups

### PR DESCRIPTION
**Pull request by S-Dafarra**
Small review of the code.

It seems that the world loads ``icubGazeboSim`` world: https://github.com/robotology/icub-gazebo-wholebody/blob/master/worlds/icub_simple_fixed_world/icub_simple_fixed.world#L23

@traversaro if you don't have any objections I would merge this.

### CODE REVIEW

**Comment by S-Dafarra** [comment]
For time constraints I will merge.

**Comment by traversaro** [comment]
Ok, no problem. Some inline comments for the feature.

File: main.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@ int main(int argc, char **argv)
 
     if (!yarp::os::Network::checkNetwork(5.0)) {
         std::cout << "Yarp network not found\n";
-        return false;
+        return 1;
```
Comment (traversaro) [inline main.cpp:13]: `EXIT_FAILURE` define in `cstdlib` is a bit more readable.

File: Module.h [hunk 2]
```diff
@@ -4,9 +4,6 @@
 #include <yarp/os/RFModule.h>
 #include <yarp/sig/Vector.h>
 
-// Workaround for https://github.com/robotology/yarp/issues/1844
-#include <yarp/os/Vocab.h>
```
Comment (traversaro) [inline Module.h:8]: To make sure that anyone using an oldish YARP gets a  clear error message, I would suggest to also increase the version of YARP requested in https://github.com/vvv-school/tutorial_impedance-control/commit/b84b2394f80454972ce2021aba571a9957ea6a1c#diff-af3b638bc2a3e6c650974192a53c7291R5 . Unfortunately I noticed that  https://github.com/robotology/yarp/issues/1844 is still not part of any released version of YARP, so there is way of checking this. Not a big problem, but it is better to remember to update  https://github.com/vvv-school/tutorial_impedance-control/commit/b84b2394f80454972ce2021aba571a9957ea6a1c#diff-af3b638bc2a3e6c650974192a53c7291R5  as soon as  YARP 3.2 or 3.1.1 gets released.

**Review by traversaro — COMMENTED** [review]
Memento.

**Pull request merged**